manifests/init.pp

```diff
@@ -188,6 +188,8 @@
 #
 # $yum_max_speed::              The maximum download speed for a Pulp task, such as a sync. (e.g. "4 Kb" or 1Gb")
 #
+# $gpg_sign_repo_metadata::     Whether repo metadata GPG signing will be enabled
```

Comment:
Slight nit: should this perhaps be ```yum_gpg_sign_repo_metadata``` or at least mention it in the documentation?

Reply:
Ok, I don't have a strong opinion, so I'll change it.